code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -418,6 +418,14 @@
 	G.key = ghost.key
 	G << "You are an adamantine golem. You move slowly, but are highly resistant to heat and cold as well as blunt trauma. You are unable to wear clothes, but can still use most tools. Serve [user], and assist them in completing their goals at any cost."
 	qdel(src)
+	var/datum/objective/protect/new_objective = new /datum/objective/protect
+	new_objective.owner = G:mind
+	new_objective:target = usr:mind
+	G.mind.objectives += new_objective
+	new_objective.explanation_text = "Protect [usr.real_name], your master."
+	if(usr.mind.special_role)
+		message_admins("[G.real_name] has been summoned by [usr.real_name], an antagonist.")
```

Comment:
Why not also tell that to the Master as well? A golem is required to protect and obey its master, so I do not think it is much of a risk.